magellan-library/src/test/java/com/wealthfront/magellan/view/DialogComponentTest.kt

```diff
@@ -29,16 +29,16 @@ class DialogComponentTest {
   @Before
   fun setUp() {
     initMocks(this)
-    dialogCreator1 = { context -> dialog1 }
-    dialogCreator2 = { context -> dialog2 }
+    dialogCreator1 = DialogCreator { context -> dialog1 }
+    dialogCreator2 = DialogCreator { context -> dialog2 }
   }
 
   @Test
   fun showDialog() {
     `when`(dialog1.isShowing).thenReturn(true)
 
     dialogComponent.create(context)
-    dialogComponent.showDialog { dialogCreator1.invoke(context) }
+    dialogComponent.showDialog(DialogCreator { context -> dialogCreator1.createDialog(context) })
```

Comment:
Um, should we unwrap this and just pass `dialogCreator1` directly?

Reply:
d